src/stare/models/confnote.py

```
@@ -68,3 +75,151 @@ class ConfNote(_Base):
     associated_analysis: RelatedPublication | None = None
     superseded_by: RelatedPublication | None = None
     phase1: ConfNotePhase1 | None = None
+
+    def __rich__(self) -> Panel:
```

Ruff (0.15.11)

[warning] Too many branches (25 > 12)

(PLR0912)